src/cloud-api-adaptor/test/e2e/docker_test.go

```diff
@@ -129,3 +129,43 @@ func TestDockerCreatePeerPodWithAuthenticatedImageWithValidCredentials(t *testin
 		t.Skip("Registry Credentials, or authenticated image name not exported")
 	}
 }
+
+func TestDockerCreateWithCpuLimit(t *testing.T) {
+	t.Parallel()
+
+	assert := DockerAssert{}
+	DoTestPodWithCpuMemLimitsAndRequests(t, testEnv, assert, "", "", "1", "")
```

Comment:
I guess related to the above comment, if we provided request limit greater than 1 what currently happens? I appreciate we'd like to drive the instance type/profile based on this in future, but I don't think that is implemented yet?

Reply:
> I guess related to the above comment, if we provided request limit greater than 1 what currently happens?
It should fail.